scss/5-objects/_grid.scss

```diff
@@ -1,3 +1,8 @@
+@import '../1-settings/breakpoints';
```

Comment:
This allows us to assert the output of grid styles, and allows grids to be imported standalone. Because we also assert that settings and tools don't output any CSS this is safe to do.